platforms-serverless/azure-functions-linux/finally.sh

```diff
@@ -1,12 +1,30 @@
 #!/bin/sh
 
+TEST_EXIT_CODE=$1
+
 set -eux
 
 app="$(cat func-tmp.txt)"
 
 group="prisma-e2e-linux"
+subscriptionId="63cbd994-4c44-4b0d-ba9f-463c6ec6ee31"
+
+echo "Too View Logs Visit"
+echo "https://portal.azure.com/#blade/WebsitesExtension/FunctionMenuBlade/monitor/resourceId/%2Fsubscriptions%2F${subscriptionId}%2FresourceGroups%2F${group}%2Fproviders%2FMicrosoft.Web%2Fsites%2F${app}%2Ffunctions%2F${app}"
```

Comment:
Will only be available if we do not delete the function and insights, so should be moved down into the  `else`.

Reply:
Moved to else